### PR TITLE
Added 'Assessment' page

### PR DESCRIPTION
**Pull request by kariljordan**
@jduckles I added an 'Assessment' page with links to the public-survey-info repo. I don't know how to make sure this shows up under the 'About' dropdown. Can you make it go? :)

### CODE REVIEW

File: pages/assessment.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Assessment
```
Comment (rgaiacs) [inline pages/assessment.md]: Need to replace this line with

~~~
---
layout: page-fullwidth
permalink: /assessment/
title: Assessment
redirect_from:
  - /pages/assessment.html
---
~~~

File: pages/assessment.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Assessment  
+
+Software Carpentry values a culture of assessment, and we are consistently evaluating our workshops to understand the impact workshops have on our learners.
+
+# Surveys
```
Comment (rgaiacs) [inline pages/assessment.md]: This line should start with `##` instead of `#`.

File: pages/assessment.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+- [Pre-workshop survey](https://www.surveymonkey.com/r/Preview/?sm=V6gQbbOKn3NoPKfYKHjAKu_2BBCdtXXsTS2pf1BIdARccEtJQqlu1KFB2j2TcF0MCn)
+
+- [Post-workshop survey](https://www.surveymonkey.com/r/Preview/?sm=uN5QPa4MbF1_2BB1plbLWnL1ZUc7Nttqici0Nc0e3G4RahMwwGW5NUp4U5PKQDYmky)
+
+Additionally, we began collecting data on the long-term impact our workshops are having on both Data Carpentry and Software Carpentry learners. The [Carpentry Long-Term Feedback Survey](https://www.surveymonkey.com/r/Preview/?sm=LksuekfCD3hzLW6lPkx9qhkRF5nDt8uGWpN7lq2Mx0Dqw1Zriv3qYFpu3XtR46ei) was launched in March 2017, and data will be collected every 6 months.
+
+- [Carpentry Long-Term Feedback Survey](https://www.surveymonkey.com/r/Preview/?sm=LksuekfCD3hzLW6lPkx9qhkRF5nDt8uGWpN7lq2Mx0Dqw1Zriv3qYFpu3XtR46ei)
+
+# Data
```
Comment (rgaiacs) [inline pages/assessment.md]: This line should start with `##` instead of `#`.

File: pages/assessment.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+# Surveys  
+We conduct pre and post workshop surveys for each of our workshops:
+
+- [Pre-workshop survey](https://www.surveymonkey.com/r/Preview/?sm=V6gQbbOKn3NoPKfYKHjAKu_2BBCdtXXsTS2pf1BIdARccEtJQqlu1KFB2j2TcF0MCn)
+
+- [Post-workshop survey](https://www.surveymonkey.com/r/Preview/?sm=uN5QPa4MbF1_2BB1plbLWnL1ZUc7Nttqici0Nc0e3G4RahMwwGW5NUp4U5PKQDYmky)
+
+Additionally, we began collecting data on the long-term impact our workshops are having on both Data Carpentry and Software Carpentry learners. The [Carpentry Long-Term Feedback Survey](https://www.surveymonkey.com/r/Preview/?sm=LksuekfCD3hzLW6lPkx9qhkRF5nDt8uGWpN7lq2Mx0Dqw1Zriv3qYFpu3XtR46ei) was launched in March 2017, and data will be collected every 6 months.
+
+- [Carpentry Long-Term Feedback Survey](https://www.surveymonkey.com/r/Preview/?sm=LksuekfCD3hzLW6lPkx9qhkRF5nDt8uGWpN7lq2Mx0Dqw1Zriv3qYFpu3XtR46ei)
```
Comment (rgaiacs) [inline pages/assessment.md]: You can remove this line since all the information it provide is in the previous paragraph.

**Review by rgaiacs — CHANGES_REQUESTED** [review]
> I don't know how to make sure this shows up under the 'About' dropdown. Can you make it go? :)

If you follow my comments you should have this page under the "About" dropdown menu at the navigation bar.

**Review by jduckles — APPROVED** [review]
I've addressed these changes.

**Review by kariljordan — COMMENTED** [review]
Thank you @jduckles! Is there something I need to do to make it appear on the website? I still don't see it.

**Comment by jduckles** [comment]
Just made the change to add it to the menu: https://github.com/swcarpentry/website/commit/cf531df6e607a85d9c7a0154dd298b4d2ded04a1

**Pull request merged**